src/parse.c

```diff
@@ -4440,7 +4440,7 @@ void Parser::checkCstyleTypeSyntax(Loc loc, Type *t, int alt, Identifier *ident)
 
 Statement *Parser::parseStatement(int flags, const utf8_t** endPtr, Loc *pEndloc)
 {
-    Statement *s;
+    Statement *s = NULL;
```

Comment:
This warning is a false positive. Initializing it to `NULL` can actually hide a bug, as all paths must initialize it. I suggest initializing it instead to `(Statement *)-1`.